contents/teams/surveys/objectives.mdx

```diff
@@ -1,61 +1,61 @@
-# ✅ Goal 1: Mobile surveys, Everywhere
+# 💰 Goal 1: Pricing
 
-We’ve already got mobile surveys working in React Native and iOS, and we’ve kicked off work for Flutter and Android [tracking here](https://github.com/PostHog/posthog/issues/25085). Next step: bring everything to feature parity with Web.
+> Why: Making surveys more accessible by revisiting our pricing model.
 
-- ✅ iOS
-- ✅ React Native
-- 🛠️ Android
-- 🛠️ Flutter
+> Who: <TeamMember name="Cory Slater" photo />, <TeamMember name="Manoel Aranda Neto" photo /> and the [Billing Team](/teams/billing).
 
-# 📈 Goal 2: Level up survey analysis
 
-You asked — we listened. We're making survey analysis more useful and enjoyable, starting with a brand new global analytics page.
+You’ve told us, and we’ve heard clearly that our pricing for high-volume surveys isn’t competitive. We’re guided by the principles outlined in our [How we make money](https://posthog.com/handbook/how-we-make-money) and [Pricing principles](https://posthog.com/handbook/engineering/feature-pricing) pages, which remind us not to let pricing be a barrier and to aim to be the most affordable option for each product. In that spirit, we’re revisiting our pricing model with a few ideas in mind, including ways to lower the cost for high-volume surveys and exploring whether single-question surveys can be made more affordable or even free.
 
-- [Global analytics view](https://github.com/PostHog/posthog/issues/28418)
-- A cleaner, more useful survey responses page
-    - [Allow customization on columns for the Survey Table Results](https://github.com/PostHog/posthog/issues/29844)
-- [Be able to see surveys result breakdown by week](https://github.com/PostHog/posthog/issues/30150)
-- [Better results analysis with metric based templates](https://github.com/PostHog/posthog/issues/19517)
-- [Don't calculate NPS score for non-NPS questions](https://github.com/PostHog/posthog/issues/19500)
-- Extra polish and UX improvements across the board, pair up with the Design team to get some ideas
+# 📣 Goal 2: Marketing
 
-# 🎨 Goal 3: Survey customization
+> Why: Increasing awareness and adoption of Surveys through focused outreach.
```

Comment:
Can we say what exactly we will be doing here? Eg
- Telling Joe something to put in the changelog, every time he asks
- Suggesting 5 new tutorials ideas for the docs team for surveys
- Understanding what ads are going out for surveys, and suggesting changes